### PR TITLE
Run HiddenEye in Docker

### PR DESCRIPTION
**Pull request by MoisesTapia**
This contribution is with docker, which are install in many sistemas like windows or all linux

Feaures:

- [X] Run HiddenEye with docker
- [X] Make image of docker with HiddenEye
- [x] Expose of container to insternet

ps
start
URL

### CODE REVIEW

**Comment by sTiKyt** [comment]
All checks failed

**Comment by sTiKyt** [comment]
Also it's outdated

**Comment by MoisesTapia** [comment]
@sTiKyt  check my code 

this one is not part of my code:

*Found in Defs\FeatureManager\EmailManager\emailconfig.py:10*

```
10 gmail_password = "GMAILPASSWORD"
```
The version of docker-compose is the most recent,

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5950029&bid=19503134)

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ WORKDIR /home/
 
 COPY . /home/
 
-RUN apt-get update && \
+RUN apt-get update -y --no-install-recommends && \
     apt-get install -y --no-install-recommends python3.7 && \
     apt-get install -y python3-pip && \
     apt-get install --no-install-recommends -y php && \
```
Comment (sTiKyt) [inline Dockerfile]: I think those "apt-get" commands won't work on Arch Linux and other distributives not using apt

Comment (MoisesTapia) [reply Dockerfile]: yeah, but I am creating my Dockerfile over Debian distribution the user don't need run those ones commands, docker allows us run the applications in most distributions and the containers are ephemeral, it allows us delete the container when we want

Comment (sTiKyt) [reply Dockerfile]: I didn't learn much about docker yet, does it download it's own distribution or WDYM?

Comment (MoisesTapia) [reply Dockerfile]: yes, docker is based in containers we can run docker in linux and windows, now withy my Pr we can run HiddenEye without install it in our native operative system

Comment (sTiKyt) [reply Dockerfile]: Okay, i see no reason to reject, approving my review

**Comment by sTiKyt** [comment]
Before i merge i want to inform you, @MoisesTapia that HiddenEye code is going to be completely redone soon. I hope you'll be here to update those docker files after next update.

**Comment by MoisesTapia** [comment]
@sTiKyt  counting on that !!

**Comment by sTiKyt** [comment]
@MoisesTapia Thank you for your contribution!

**Pull request merged**